src/Centreon/Domain/Entity/Topology.php

```diff
@@ -304,6 +316,22 @@ public function setTopologyShow(string $topology_show): void
         $this->topology_show = $topology_show;
     }
 
+    /**
+     * @return string
+     */
+    public function getIsDeprecated(): ?string
+    {
+        return $this->is_deprecated;
+    }
+
+    /**
+     * @param string $isDeprecated
+     */
+    public function setIsDeprecated(string $isDeprecated): void
```

Comment:
```suggestion
    public function setDeprecated(bool $isDeprecated): void
```

Reply:
It is the legacy code. I can't change it cause it breaks entity creation